cmd/comment/update.go

```diff
@@ -0,0 +1,69 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package cmdcomment
+
+import (
+	"path/filepath"
+
+	"github.com/elastic/cloud-sdk-go/pkg/api/commentapi"
+	"github.com/spf13/cobra"
+
+	"github.com/elastic/ecctl/pkg/ecctl"
+)
+
+var updateCmd = &cobra.Command{
+	Use:     "update <comment id> <message> --resource-type <resource-type> --resource-id <resource-id>",
+	Short:   "Updates an existing resource comment",
+	PreRunE: cobra.ExactValidArgs(2),
```

Comment:
I think you want to use `cobra.ExactArgs` since there isn't  `ValidArgs` set in this command.
```suggestion
	PreRunE: cobra.ExactArgs(2),
```